.gitignore

```diff
@@ -4,5 +4,7 @@ tmp/
 .vs/
 .idea/
 
+proto/googleapis
```

Comment:
I wouldn't expect this to be necessary - git knows what a submodule is, and shouldn't be trying to add it as a normal file.

Reply:
There should be no cloning of googleapis, this was never used as part of the `protoc` generation process and was a mistake adding it. This instruction is outdated.